packages/gatsby/src/bootstrap/index.js

```diff
@@ -82,24 +83,6 @@ module.exports = async (args: BootstrapArgs) => {
     payload: program,
   })
 
-  let activityForJobs
-
-  emitter.on(`CREATE_JOB`, () => {
-    if (!activityForJobs) {
-      activityForJobs = report.phantomActivity(`Running jobs`)
-      activityForJobs.start()
-    }
-  })
-
-  const onEndJob = () => {
-    if (activityForJobs && store.getState().jobs.active.length === 0) {
-      activityForJobs.end()
-      activityForJobs = null
-    }
-  }
-
-  emitter.on(`END_JOB`, onEndJob)
-
```

Comment:
Should we keep that? I think with this removed v1 jobs won't have activity anymore is it's only created in `enqueueJob` called by `createJobV2` action creator (if we keep this, we probably should have different activity names for v1 jobs in v2 jobs)

Reply:
oh yeah you're right, we probably still need this to keep older versions of the jobs api to not break structured logging, let me revert.